packages/common/client-utils/src/events/emitter.ts

```diff
@@ -146,6 +146,29 @@ export class CustomEventEmitter<TListeners extends Listeners<TListeners>>
 		return () => this.off(eventName, listener);
 	}
 
+	public once<K extends keyof Listeners<TListeners>>(
+		eventName: K,
+		listener: TListeners[K],
+	): void {
```

Comment:
this should still return off, like on does, as once doesn't mean always once, it means at most once